scene/gui/dialogs.cpp

```diff
@@ -224,7 +226,7 @@ void AcceptDialog::_update_child_rects() {
 
 	for (int i = 0; i < get_child_count(); i++) {
 		Control *c = Object::cast_to<Control>(get_child(i));
-		if (!c) {
+		if (!c || !c->is_visible()) {
```

Comment:
And this.
Unless it's related to this PR, it should be reverted/extracted to new PR.